scss/components/_banner.scss

```diff
@@ -1,5 +1,16 @@
 [role=banner] {
   ul {
     display: flex;
+    padding: 9px;
+    border: 1px solid black;
+    background-color: blue;
+  }
+  ul li a {
```

Comment:
Since this is SCSS, you can move this inside the `ul` declaration above it.

```
ul {
  display: flex;
  padding: 9px;
  border: 1px solid black;
  background-color: blue;

  li a {  
    text-decoration: none;
    padding: 5px;
    margin: 5px;
    color: blue;
    background-color: white;
    border-radius: 3px;
  }
}
```

Reply:
oh